### PR TITLE
push chat message.

### PR DESCRIPTION
**Pull request by lyw0149**
make sendbird router and push message.

### CODE REVIEW

File: src/server/index.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import methodOverride from 'method-override';
 import users from './routes/users.route';
 import survey from './routes/survey.route';
 import match from './routes/match.route';
+import chat from './routes/chat.route';
```
Comment (KyuaKwon) [inline src/server/index.js]: Alphabetical order is needed

File: src/server/routes/chat.route.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import * as pushUtil from '../utils/push.util';
+
+const router = express.Router();
+
+router.post('/callback', (req, res, next)=> {
```
Comment (KyuaKwon) [inline src/server/routes/chat.route.js]:
```
... (req, res, next) => {
```

Comment (KyuaKwon) [inline src/server/routes/chat.route.js]: Does only one method is needed in chat.route.js?
Then just making url as ```'/'``` would be one of the way we can do.

Comment (lyw0149) [inline src/server/routes/chat.route.js]: Get api key route will be added.

File: src/server/utils/push.util.js [hunk 3]
```diff
@@ -35,23 +35,17 @@ export function sendPush(receiverId, notificationType, bodyParam) {
...
         };
         const fcm = new FCM(serverKey);
-        fcm.send(message)
-          .then(function (response) {
-            console.log(response);
-          })
-          .catch(function (err) {
-            console.log(err);
-          });
+        fcm.send(message);
```
Comment (SarahhhC) [inline src/server/utils/push.util.js]: Nice!

**Comment by SarahhhC** [comment]
LGTM Thanks @lyw0149 👍

**Pull request merged**